scss/civicrm/contact/pages/_tag-contacts.scss

```diff
@@ -109,64 +109,83 @@
   }
 }
 
-.CRM_Tag_Form_Tag {
-  #Tag {
+.CRM_Tag_Form_Tag #Tag,
+.tag-tree-wrapper {
+
+  table {
     background: $crm-white;
+    border: none;
     box-shadow: $box-shadow;
 
-    h3 {
-      background: $gray-lighter;
-      color: $gray-darker;
-      font-size: $font-size-h2;
-      font-family: $font-family-base;
-      vertical-align: middle;
+    thead th {
       border: none;
-      padding: 17px 20px;
       border-bottom: 1px solid $crm-dark-grayblue;
+      font-family: $font-family-base;
+      font-size: $font-size-h2;
+      font-weight: $crm-font-weight-h3;
+      padding: 17px 20px;
+      vertical-align: middle;
     }
 
-    #tagtree {
+    tbody td{
+      border: none;
       padding: 20px;
+    }
+  }
+
+  .crm-form-text {
+    @extend %custom-form-control, %open-sans-font-family;
+    display: inline-block;
+    width: 230px;
+  }
+
+  #tagtree,
+  .tag-tree {
+    margin-left: -10px;
+
+    li {
+      margin: 10px;
+
+      div.jstree-wholerow-hovered,
```

Comment:
Do we have a CiviCRM class called jstree-wholerow-hovered? or is it generated by external library/ extension?